src/api/procedures/consumeAuthorizationRequests.ts

```diff
@@ -111,26 +111,22 @@ export async function getAuthorization(
 ): Promise<ProcedureAuthorization> {
   const { context } = this;
 
-  let did: string;
+  const identity = await context.getCurrentIdentity();
```

Comment:
The idea behind `fetchDid` was to avoid making the call to the chain if not necessary. We should preserve that optimization I think